spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -47,19 +47,155 @@ definitions:
+      be identified by the index.
+    fields:
+    - time:
+        type: GPSTime
+        desc: GNSS time of the STEC data
+    - num_msgs:
+        type: u16
+        desc: Number of messages in the dataset
+    - seq_num:
+        type: u16
+        desc: Position of this message in the dataset
+    - ssr_update_interval:
+        type: u16
+        units: seconds
+        desc: update interval in seconds
+    - iod_ssr:
+        type: u8
+        desc: range 0 - 15
+    - tropo_quality:
+        type: u8
+        units: encoded value - see spec for details
+        desc: troposphere quality indicator
+
+ - STECSatElement:
+     desc: STEC for the given satellite.
+     fields:
+     - sv_id:
+         type: SvId
+         desc: Unique space vehicle identifier
+     - stec_quality_indicator:
+         type: u8
+         units: encoded value - see spec for details
+         desc: quality of STEC data
+     - stec_coeff:
+         type: array
+         size: 4
+         fill: s16
+         desc: coefficents of the STEC polynomial
+
+ - TroposphericDelayCorrection:
+    short_desc: troposphere delay correction
+    desc: |
+      Contains wet vertical and hydrostatic vertical delay
+    fields:
+    - hydro:
+        type: s16
+        units: 0.4 mm (add 2.3 m to get actual vert hydro delay)
+       desc: hydrostatic vertical delay
+    - wet:
+        type: s8
+        units: 0.4 mm (add 0.252 m to get actual vert wet delay)
+        desc: wet vertical delay
+
+ - STECResidual:
+    desc: STEC residual
+    fields:
+    - sv_id:
+        type: SvId
+        desc: space vehicle identifier
+    - residual:
+        type: s16
+        desc: STEC residual (Scale factor 0.04 TECU)
+
+ - GridElement:
+    short_desc: Grid datum for troposphere and STEC residuals
+    desc: |
+      Contains one tropo datum, plus STEC residuals for each space
+      vehicle
+    fields:
+    - index:
+        type: u16
+        desc: index of the grid point
+    - tropo_delay_correction:
+        type: TroposphericDelayCorrection
+        desc: Wet and Hydrostatic Vertical Delay
+    - STEC_residuals:
+        type: array
+        fill: STECResidual
+        desc: STEC Residual for the given space vehicle
+
+ - GridDefinitionHeader:
+    short_desc: Defines the grid for STEC and tropo grid messages
+    desc: |
+      Defines the grid for STEC and tropo grid messages.
+      Also includes an RLE encoded validity list.
+    fields:
+    - region_size_inverse:
+        type: u8
+        units: coded value - see spec
+        desc: inverse of region size
+    - area_width:
+        type: u16
+        units: coded value - 0 means not present (Swift convention)
+        desc: area width; see spec for details
+    - lat_nw_corner_enc:
+        type: u16
+        desc: encoded latitude of the northwest corner of the grid
+    - lon_nw_corner_enc:
+        type: u16
+        desc: encoded longitude of the northwest corner of the grid
+    - num_msgs:
```

Comment:
Will there be more than 1 GridDefinitionHeader messages?

Reply:
that was done to support the RLE validity list.  I think that thing is pretty janky.  Why would we ever send invalid data?  I'd be more than happy to snip that whole RLE list out of the message, and if we really need it, include validity flag in the correction/STEC messages.  Thoughts?

Reply:
I think I see where they are going with this;  because it's broadcast, everyone is consuming it.  If the rover is in some corner case, we might want to say "hey, don't use this, because you are out of bounds from the model/whatever"

Reply:
It would be a minor deviation from spec, but I could chop out the multi message support, since I think is it unlikely we'll see an RLE list that long, and you said these aren't prod messages.

Reply:
Yeah I forgot about the RLE validity list, let's keep it.